### DISCUSSION TITLE
D4 corrections for superheavy elements ?

### DISCUSSION THREAD

**Discussion by miroi** [dftd4/dftd4#244] [Ideas]
Hello,

we are working in the field of superheavy elements (Z up to 120). 

I would like to ask you how difficult it will be to implement D4 corrections (also D3(BJ) are wanted) for superheavy elements. Thanks.

**Comment by marvinfriede** [comment]
We recently extended both D3 and D4 to the full actinide series (including Fr and Ra). We will create new releases for the D3 and D4 standalone programs as soon as the corresponding paper is published (currently in review). The extension will also be part of ORCA 6.0.0. Further extensions are not planned from our side. However, the procedure (to generate the C6 coefficients) is described in detail in the upcoming paper.

(Related discussion in https://github.com/dftd4/dftd4/issues/204.)

cc @lukaswittmann @maburs

**Reply by miroi** [reply]
Thanks. Looking forward for your paper to appear.

**Reply by marvinfriede** [reply]
The paper is now published: [10.1039/D4CP01514B](https://pubs.rsc.org/en/content/articlelanding/2024/cp/d4cp01514b).

**Comment by miroi** [comment]
Thanks, closing.
